internal/distro/rhel84/distro.go

```diff
@@ -218,6 +218,19 @@ func (t *imageType) Packages(bp blueprint.Blueprint) ([]string, []string) {
 		packages = removePackage(packages, "insights-client")
 	}
 
+	// copy the list of excluded packages from the image type
+	// and subtract any packages found in the blueprint (this
+	// will not handle the issue with dependencies present in
+	// the list of excluded packages, but it will create a
+	// possibility of a workaround at least)
+	excludedPackages := append([]string(nil), t.excludedPackages...)
+	for _, pkg := range bp.GetPackages() {
+		// removePackage is fine if the package doesn't exist
+		excludedPackages = removePackage(excludedPackages, pkg)
+	}
```

Comment:
Had an immediate reaction to this nested iteration here.  I guess blueprints don't tend to have hundreds of packages listed in them, and we don't exclude more than a handful, so in the end it's fine.

Reply:
Exactly and Go is said to be fast, right? :-D

Reply:
Yeah, Go is so fast that it can do `O(n^2)` in `O(n)` :-D /s